app/src/lib/app-state.ts

```diff
@@ -39,6 +39,7 @@ import { IStashEntry } from '../models/stash-entry'
 import { TutorialStep } from '../models/tutorial-step'
 import { UncommittedChangesStrategy } from '../models/uncommitted-changes-strategy'
 import { CherryPickFlowStep } from '../models/cherry-pick'
+import { DragElement } from '../models/dragElement'
```

Comment:
I don't think we have any file in the code base with camel casing, instead I think it should be…
```suggestion
import { DragElement } from '../models/drag-element'
```